pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -105,7 +106,8 @@ public class IndexLoadingConfig {
    */
   public IndexLoadingConfig(InstanceDataManagerConfig instanceDataManagerConfig, TableConfig tableConfig,
       @Nullable Schema schema) {
-    extractFromInstanceConfig(instanceDataManagerConfig);
+    _instanceDataManagerConfig = instanceDataManagerConfig;
+    extractFromInstanceConfig();
```

Comment:
(minor) Let's make it consistent as how table config and schema is handled (set `_instanceDataManagerConfig` within `extractFromInstanceConfig(instanceDataManagerConfig)`)